diff --git a/languages.toml b/languages.toml
index a5403e6..e0f09d2 100644
--- a/languages.toml
+++ b/languages.toml
@@ -13,6 +13,7 @@ ameba-ls = { command = "ameba-ls" }
 angular = {command = "ngserver", args = ["--stdio", "--tsProbeLocations", ".", "--ngProbeLocations", ".",]}
 asm-lsp = { command = "asm-lsp" }
 awk-language-server = { command = "awk-language-server" }
+ayame-spell = { command = "ayame-spell", args = ["lsp", "--stdio"] }
 bash-language-server = { command = "bash-language-server", args = ["start"] }
 bass = { command = "bass", args = ["--lsp"] }
 beancount-language-server = { command = "beancount-language-server" }
